waku/incentivization/reputation_manager.nim

```diff
@@ -28,14 +28,16 @@ proc getReputation*(manager: ReputationManager, peer: PeerId): Option[bool] =
   else:
     result = none(bool)
 
+### Lightpush-specific functionality ###
+
 # Evaluate the quality of a LightPushResponse by checking its status code
 proc evaluateResponse*(response: LightPushResponse): ResponseQuality =
   if response.statusCode == LightpushStatusCode.SUCCESS.uint32:
     return GoodResponse
   else:
     return BadResponse
```

Comment:
We'll need a more generic way of doing this, eventually.

Reply:
Sure. By generic you mean supporting evaluation of responses of other protocols?

Reply:
> Sure. By generic you mean supporting evaluation of responses of other protocols?

Yes and also avoid importing types from other part of the code.